presto-main/src/test/java/io/prestosql/operator/TestDynamicFilterSourceOperator.java

```diff
@@ -277,63 +394,105 @@ public void testCollectEmptyBuildSide()
     }
 
     @Test
-    public void testCollectTooMuchRows()
+    public void testCollectTooMuchRowsSingleColumn()
     {
-        int maxRowCount = getDynamicFilteringMaxPerDriverRowCount(pipelineContext.getSession());
-        Page largePage = createSequencePage(ImmutableList.of(BIGINT), maxRowCount + 1);
+        int maxPositionsCount = 100;
+        Page largePage = new Page(createLongSequenceBlock(0, maxPositionsCount + 1));
 
-        OperatorFactory operatorFactory = createOperatorFactory(channel(0, BIGINT));
-        verifyPassthrough(createOperator(operatorFactory),
-                ImmutableList.of(BIGINT),
-                largePage);
-        operatorFactory.noMoreOperators();
-        assertEquals(partitions.build(), ImmutableList.of(TupleDomain.all()));
+        TupleDomain<DynamicFilterId> expectedTupleDomain = TupleDomain.withColumnDomains(ImmutableMap.of(
+                new DynamicFilterId("0"), Domain.create(ValueSet.ofRanges(
+                        range(BIGINT, 0L, true, (long) maxPositionsCount, true)), false)));
+        assertDynamicFilters(maxPositionsCount, ImmutableList.of(expectedTupleDomain), ImmutableList.of(BIGINT), largePage);
     }
 
     @Test
-    public void testCollectTooMuchBytesSingleColumn()
+    public void testCollectTooMuchRowsMultipleColumns()
```

Comment:
rename to ``testMultipleColumnsCollectMinMaxRangeWhenTooManyPositions``

Reply:
done